src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -194,6 +194,7 @@ export class SettingsEditor2 extends EditorPane {
 	private _searchResultModel: SearchResultModel | null = null;
 	private searchResultLabel: string | null = null;
 	private lastSyncedLabel: string | null = null;
+	private settingsIndex: Map<string, number> | null = null;
```

Comment:
Do you want to give this a name that describes what is being indexed? I have to read the code to find out

Reply:
Will rename in another PR